### ISSUE TITLE
release-2.0.0 add contact doesn't have buttons for submit or cancel

### ISSUE THREAD

**Issue by jbmanos** [mautic/mautic#1942]
## Description
Going to the Contacts tab on php 7.0.8 installation and clicking the "add new" button results in the screen with core and social options, but no buttons to save or cancel appear.  So no new contact can be added with this method.  Using Quick Add works.

Same behavior on mac safari and firefox latest updates.

## Log Error
[2016-07-04 00:38:06] mautic.ERROR: Deprecation: Accessing type "submit" by its string name is deprecated since version 2.8 and will be removed in 3.0. Use the fully-qualified type class name "Symfony\Component\Form\Extension\Core\Type\SubmitType" instead. - in file /mauticpathl/vendor/symfony/form/FormRegistry.php - at line 100

**Comment by jbmanos** [comment]
installed from the amazon download, still no save and close buttons.  safari browser console is chock full of fraola errors.

`froala_editor.js:552SyntaxError: Unexpected string literal 'use strict'
align.js:3:88TypeError: undefined is not an object (evaluating '$.FE.PLUGINS')
(anonymous function) — align.js:3:88
(anonymous function) — align.js:3
global code — align.js:3
code_beautifier.js:3:88TypeError: undefined is not an object (evaluating '$.FE.PLUGINS')
code_view.js:3:97TypeError: undefined is not an object (evaluating '$.FE.DEFAULTS')
colors.js:3:97TypeError: undefined is not an object (evaluating '$.FE.POPUP_TEMPLATES')
font_family.js:3:97TypeError: undefined is not an object (evaluating '$.FE.DEFAULTS')
font_size.js:3:97TypeError: undefined is not an object (evaluating '$.FE.DEFAULTS')
fullscreen.js:3:88TypeError: undefined is not an object (evaluating '$.FE.PLUGINS')
image.js:3:97TypeError: undefined is not an object (evaluating '$.FE.POPUP_TEMPLATES')
filemanager.js:6:88TypeError: undefined is not an object (evaluating '$.FE.PLUGINS')
inline_style.js:3:97TypeError: undefined is not an object (evaluating '$.FE.DEFAULTS')
line_breaker.js:21TypeError: undefined is not an object (evaluating '$.FE.DEFAULTS')
link.js:3:97TypeError: undefined is not an object (evaluating '$.FE.POPUP_TEMPLATES')
lists.js:3:88TypeError: undefined is not an object (evaluating '$.FE.PLUGINS')
paragraph_format.js:3:97TypeError: undefined is not an object (evaluating '$.FE.DEFAULTS')
paragraph_style.js:3:97TypeError: undefined is not an object (evaluating '$.FE.DEFAULTS')
quick_insert.js:3:97TypeError: undefined is not an object (evaluating '$.FE.DEFAULTS')
quote.js:3:88TypeError: undefined is not an object (evaluating '$.FE.PLUGINS')
table.js:3:97TypeError: undefined is not an object (evaluating '$.FE.POPUP_TEMPLATES')
url.js:3:97TypeError: undefined is not an object (evaluating '$.FE.DEFAULTS')
video.js:3:97TypeError: undefined is not an object (evaluating '$.FE.POPUP_TEMPLATES')
token.js:3:88TypeError: undefined is not an object (evaluating '$.FE.PLUGINS')
activateGlobalFroalaOptions — app.js:13:946TypeError: undefined is not an object (evaluating 'mQuery.FroalaEditor.ENTER_BR')
activateGlobalFroalaOptions — app.js:13:946TypeError: undefined is not an object (evaluating 'mQuery.FroalaEditor.ENTER_BR')
activateGlobalFroalaOptions — app.js:13:946
onPageLoad — app.js:13:3764
processPageContent — app.js:73
success — app.js:47
fire — libraries.js:303:305
fireWith — libraries.js:308:203
done — libraries.js:628:107
(anonymous function) — libraries.js:645:224
Selected Element
<div class="pt-md pr-md pl-md pb-md">…</div>
 = $1
Selected Element
<div class="pt-md pr-md pl-md pb-md">…</div>
 = $1`

**Comment by escopecz** [comment]
Please, could you confirm you are having (or not having) this issue with production package of Mautic 2.0.0?

**Comment by jbmanos** [comment]
Yes.   Same problem, but what is strange is that the errors are only when accessing from a desktop or laptop.   When using an iPad to access the site, the graphs appears and all is well.  

Desktop browsers were safari and Firefox    Behavior was the same on both.

**Comment by escopecz** [comment]
So the issue is there only when displaying the contact form on mobile devices?

**Comment by escopecz** [comment]
I emulated several mobile devices and the buttons are still there.

**Comment by tiagosh** [comment]
I was having the very same problem, but the entire system was behaving weird after the upgrade to 2.1.0. I could not see any stats/charts in the dashboard nor add a new widget, campaign builder was also broken and the progress bar continuously running.

I manually edited froala_editor.js and removed the two 'use strict' lines and then mautic started to behave normally again. I did not dig into the problem, but it seems to me like it was caused by the js minification process.
